### ISSUE TITLE
Micropurchase Handbook Page

### ISSUE THREAD

**Issue by melanienleopold** [18F/tts-tech-portfolio#1009]
## Background information

The micropurchase handbook page needs to be updated with the new process established with OA, the Ops team, and the Tech Portfolio.

## Acceptance criteria

- [ ] The micropurchase page is updated with the new process.
- [ ] Thumbs-up from Acquisition
- [ ] Thumbs-up from BizOps

---

**Comment by melanienleopold** [comment]
Moving to in progress per Brittney's request.

**Comment by melanienleopold** [comment]
[Suggested edits ](https://docs.google.com/document/d/1QCNOLRqgnYnBzvHr4atGjnUbw4MG0uwWX4KMLb2OmaM/edit#heading=h.6hv6d6a3p7hh) are finished. Waiting on the thumbs up from BizOps and OA. It will go live once the process is updated.

**Comment by melanienleopold** [comment]
Adding 889 information.

**Comment by melanienleopold** [comment]
Waiting feedback from Biz Ops and Acquisition.

**Comment by melanienleopold** [comment]
Blocked until micropurchase sheet changes are made

**Comment by melanienleopold** [comment]
Closing in favor of [#1229](https://github.com/18F/tts-tech-portfolio/issues/1229)
